ProcessLib/BoundaryCondition/NeumannBoundaryConditionLocalAssembler.h

```diff
@@ -42,30 +44,34 @@ class UniformNeumannBoundaryConditionLocalAssembler final
                   double const t, const GlobalVector& /*x*/,
                   GlobalMatrix& /*K*/, GlobalVector& b) override
     {
-        (void)t;  // TODO time-dependent Neumann BCs
-
         _local_rhs.setZero();
 
         IntegrationMethod integration_method(Base::_integration_order);
         std::size_t const n_integration_points =
             integration_method.getNumberOfPoints();
 
-        for (std::size_t ip(0); ip < n_integration_points; ip++) {
+        SpatialPosition pos;
+        pos.setElementID(id);
+
+        for (std::size_t ip(0); ip < n_integration_points; ip++)
+        {
+            pos.setIntegrationPoint(ip);
             auto const& sm = Base::_shape_matrices[ip];
             auto const& wp = integration_method.getWeightedPoint(ip);
-            _local_rhs.noalias() +=
-                sm.N * _neumann_bc_value * sm.detJ * wp.getWeight();
+            _local_rhs.noalias() += sm.N *
+                                    _neumann_bc_value.getTuple(t, pos).front() *
```

Comment:
OK, I see that now (after starting to rebase my mechanics branch).
The question came up, because it is not obvious, reading the lines around the access, that it has exactly one component.
I would like to add Parameter::operator(t, pos) returning the only one component (assert it).
Then the name `...value` would make much more sense, when reading `value(t, pos)`...